src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -76,10 +84,12 @@ public CommandBase driveDistance(double distance, double speed) {
             double avg = (leftMotorGroup.getEncoder().getDistance() + rightMotorGroup.getEncoder().getDistance()) / 2;
             return (avg >= distance);
         }, this);
+        cmd.setName("Drive Distane");
```

Comment:
Spelling is hard

Reply:
Yikes Ben Can't Spell at 12 at night

Reply:
If your Ben at Midnight then yes, yes it is